src/components/ApplicationDownloadLink.js

```diff
@@ -1,8 +1,11 @@
 import { Page, Document, StyleSheet, View, /*PDFViewer*/ PDFDownloadLink, Text} from '@react-pdf/renderer';
 import {h, span, i} from 'react-hyperscript-helpers';
 import {DataUseTranslation} from '../libs/dataUseTranslation';
-import isEmpty from 'lodash/fp/isEmpty';
+import {find, isEmpty} from 'lodash/fp';
 import { Theme } from '../libs/theme';
+import { isNil }from "lodash";
```

Comment:
We should try to be consistent and pull `isNil` from `lodash/fp`. When in doubt, prefer the fp version.